core/metacling/src/TClingCallFunc.cxx

```diff
@@ -1046,9 +1042,16 @@ void TClingCallFunc::make_narg_call_with_return(const unsigned N, const string &
 
 tcling_callfunc_Wrapper_t TClingCallFunc::make_wrapper()
 {
+   static map<const Decl *, void *> gWrapperStore;
```

Comment:
```suggestion
   static std::unordered_map<const Decl *, void *> gWrapperStore;
```
I don't think we need the ordering guarantees.

Reply:
Let's have that in a separate change and move forward with the current state.

Reply:
Can you elaborate why? The declarations are moved anyway, so changing the type or at least prefixing it with `std::` to reduce `using` declarations comes more or less for free...

I'd also like to re-iterate on my request in https://github.com/root-project/root/pull/12745#pullrequestreview-2636498052:
> The old version had three individual commits, can we please have this state back instead of one squashed commit that touches literally hundred lines?

Reply:
> Changing the type of a map is somewhat a functional change especially when debugging callfunc wrappers and so on.

In a map used for caching? That would be very weird...